packages/components/src/illustration/src/Illustration.tsx

```diff
@@ -31,19 +30,10 @@ export function InnerIllustration({
 }: InnerIllustrationProps) {
     const orientationValue = useResponsiveValue(orientation);
 
-    const { content, heading, image } = useSlots(children, useMemo(() => ({
+    const { image } = useSlots(children, useMemo(() => ({
```

Comment:
An Illustration component shouldn't use "slots" anymore since it now only accept a single children.